CTRP.R

```diff
@@ -0,0 +1,60 @@
+library(dplyr)
+
+AUC_1 <- read.csv("AUC_1.csv", header = TRUE, stringsAsFactors = FALSE)
+AUC_2 <- read.csv("AUC_2.csv", header = TRUE, stringsAsFactors = FALSE)
+cancercellline <- read.csv("cancer_cell_line_info.csv", header = TRUE, stringsAsFactors = FALSE)
+compound <- read.csv("compound_info.csv", header = TRUE, stringsAsFactors = FALSE)
+experiment <- read.csv("Experiment_info.csv", header = TRUE, stringsAsFactors = FALSE)
+
```

Comment:
I did not set strings as factors to false when i read in my data, but it did not seem to affect it?

Reply:
Yeah, during the MBL, they told us to always include that command; in most cases without it, is perfectly fine, but in some cases, without strings as factors may cause some problems. And I totally forgot what are "some cases"... lol so I just include that everywhere whatever...

Reply:
Strings as factors got an update recently where you can set a global default to have stringsasfactors = T or F, and with this update the default for this global value is actually now that it is FALSE. But before this was absolutely true and often an issue.